ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -188,12 +174,7 @@ export class TransactionDetailsScreen extends React.Component<Props, never> {
               </Col>
             </Row>
             <Row style={styles.rowStyle}>
-              <Button
-                style={{ marginTop: 20 }}
-                block={true}
-                success={true}
-                onPress={(): boolean => navigate("")}
-              >
+              <Button style={{ marginTop: 20 }} block={true} success={true}>
```

Comment:
no inline styles, is this a special margin for this component? if not, it should come from the theme variables

Reply:
Yap that was some old stuff that came back haunting me :D Btw I believe [no-inline-styles](https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/no-inline-styles.md) or something along the lines of that may come in handy, particularly for new contributors!